packages/tailwind/src/tailwind.tsx

```diff
@@ -1,104 +1,162 @@
+/* eslint-disable @typescript-eslint/no-unsafe-member-access */
+/* eslint-disable @typescript-eslint/no-unsafe-argument */
 import * as React from "react";
 import { renderToStaticMarkup } from "react-dom/server";
-import htmlParser, {
-  attributesToProps,
-  domToReact,
-  Element,
-} from "html-react-parser";
-import { tailwindToCSS, type TailwindConfig } from "tw-to-css";
-import { cleanCss, makeCssMap, getMediaQueryCss } from "./utils";
+import type { TailwindConfig, twi as TwiType } from "tw-to-css";
+import { tailwindToCSS } from "tw-to-css";
+import { cssToJsxStyle } from "./utils/css-to-jsx-style";
 
 export interface TailwindProps {
   children: React.ReactNode;
   config?: TailwindConfig;
 }
 
+function processElement(
```

Comment:
it seems like the new process method behaves differently then the previous parser. Haven't debug deeper but after upgrade to the latest version, some Tailwind class no longer parsed.

Will try to provide a code sandbox to reproduce the issue

| **Before** | **After** |
| -- | -- |
| <img width="1251" alt="Screenshot 2023-10-31 at 11 53 31 PM" src="https://github.com/resendlabs/react-email/assets/3906130/000f9a67-eee4-47d6-b526-a6cfa9c24864"> | <img width="1252" alt="Screenshot 2023-10-31 at 11 53 22 PM" src="https://github.com/resendlabs/react-email/assets/3906130/78fbac51-a146-4cdf-939c-ad34d5cb8bfb"> |

Reply:
+1, I tried to bump `react-email` versions last night so I could upgrade my project to Next 14, and that appears to have broken all my emails, which are all wrapped with a `<Tailwind>` wrapper (padding, background, text color set with tailwind classes are the obvious things not working, not sure exactly why yet)

Agree that `"@react-email/tailwind": "0.0.9",` appears to be the last working version

Reply:
Although, I should say I'm not sure exactly what is/isn't working. E.g. the following, running with Next14 is producing the same broken Tailwind email components for me
```
    "@react-email/components": "0.0.11",
    "@react-email/render": "0.0.9",
    "@react-email/tailwind": "0.0.9",
```